seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/source/HbaseSourceReader.java

```diff
@@ -39,22 +40,21 @@
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Deque;
-import java.util.LinkedHashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
-import java.util.Set;
 import java.util.concurrent.ConcurrentLinkedDeque;
+import java.util.stream.Collectors;
 
 @Slf4j
 public class HbaseSourceReader implements SourceReader<SeaTunnelRow, HbaseSourceSplit> {
-    private static final String ROW_KEY = "rowkey";
+    private final String ROW_KEY = "rowkey";
```

Comment:
Why remove static?

Reply:
> Why remove static?

The previous methods using ROW_KEY were all non-static methods. Using this.ROW_KEY to call private final static String ROW_KEY = "rowkey"; is not a good choice.

Here is how to use ROW_KEY in historical code：
![image](https://github.com/apache/seatunnel/assets/29418975/e56abcb6-be10-4ff4-ad15-91624b93fe7b)
![image](https://github.com/apache/seatunnel/assets/29418975/80b6060f-db4f-407f-83ce-5c11b0c32d03)